[2023-01-16T03:55:30.821+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-01-16T03:55:23.350693+00:00 [queued]>
[2023-01-16T03:55:30.832+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2023-01-16T03:55:23.350693+00:00 [queued]>
[2023-01-16T03:55:30.833+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T03:55:30.835+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-16T03:55:30.835+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-16T03:55:30.851+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-01-16 03:55:23.350693+00:00
[2023-01-16T03:55:30.856+0000] {standard_task_runner.py:55} INFO - Started process 4269 to run task
[2023-01-16T03:55:30.859+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2023-01-16T03:55:23.350693+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp7n6zbs2f']
[2023-01-16T03:55:30.862+0000] {standard_task_runner.py:83} INFO - Job 48: Subtask store_user
[2023-01-16T03:55:30.925+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user manual__2023-01-16T03:55:23.350693+00:00 [running]> on host c4d6e9176664
[2023-01-16T03:55:30.992+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-01-16T03:55:23.350693+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-01-16T03:55:23.350693+00:00
[2023-01-16T03:55:30.993+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/process_user.csv
[2023-01-16T03:55:31.004+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-01-16T03:55:31.011+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-16T03:55:31.021+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20230116T035523, start_date=20230116T035530, end_date=20230116T035531
[2023-01-16T03:55:31.073+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-16T03:55:31.092+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
